hphp/hack/src/errors/errors.ml

```diff
@@ -4758,9 +4758,15 @@ let override_no_default_typeconst pos_child pos_parent =
 let inout_annotation_missing pos1 pos2 =
   let msg1 = (pos1, "This argument should be annotated with `inout`") in
   let msg2 = (pos2, "Because this is an `inout` parameter") in
-  add_list (Typing.err_code Typing.InoutAnnotationMissing) msg1 [msg2]
-
-let inout_annotation_unexpected pos1 pos2 pos2_is_variadic =
+  let (_, start_column) = Pos.line_column pos1 in
+  let pos = Pos.set_col_end start_column pos1 in
+  add_list 
+    (Typing.err_code Typing.InoutAnnotationMissing) 
+    ~quickfixes:[Quickfix.make ~title:("Insert inout type annotation") ~new_text:("inout ") pos]
```

Comment:
`inout` isn't really about types, it's about calling functions in a way they can mutate arguments.

Hack docs: https://docs.hhvm.com/hack/functions/inout-parameters
There's a few other languages with this features too, such as Swift: https://docs.hhvm.com/hack/functions/inout-parameters

```suggestion
    ~quickfixes:[Quickfix.make ~title:("Insert `inout` annotation") ~new_text:("inout ") pos]
```